### DISCUSSION TITLE
Is there a way to customize the size of qr code on the terminal?

### DISCUSSION THREAD

**Discussion by rubslopes** [3mora2/WPP_Whatsapp#40] [Q&A]
When I use headless browser and show the qr code in the terminal, the image is too large. Is there a way to customize the size of qr code on the terminal?

I'm using:
`creator = Create(session=your_session_name, browser='firefox', headless=True, catchQR=catchQR, logQR=True, qr='terminal')`
where `catchQR` comes from https://github.com/3mora2/WPP_Whatsapp/blob/main/examples/get_qrcode.py

**Comment by 3mora2** [comment]
use `urlCode` to create qrcode
Iam use
```python
def asciiQr(code):
    import segno
    import io
    qrcode = segno.make(code)
    out = io.StringIO()
    # qrcode.terminal(out=out, border=1, compact=True)
    qrcode.terminal(out=out, border=1)
    out.seek(0)
    result = out.read()
    return result

urlCode = ""
print(asciiQr(urlCode))
```

can't find other lib that print qrcode smaller
